lib/DDG/Spice/SeatGeek/EventsNearMe.pm

```diff
@@ -0,0 +1,40 @@
+package DDG::Spice::SeatGeek::EventsNearMe;
+# ABSTRACT: Returns upcoming concerts based on geolocation
+
+use DDG::Spice;
+
+primary_example_queries "live shows near me", "upcoming concerts in my area";
+description "Upcoming concerts from SeatGeek";
+name "SeatGeek Events By Geolocation";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/SeatGeek/EventsNearMe.pm";
+category "entertainment";
+topics "entertainment", "music";
+attribution github => ['https://github.com/MariagraziaAlastra','MariagraziaAlastra'],
+    github => ['https://github.com/andrey-p','Andrey Pissantchev'];
+
+triggers start => 'upcoming concerts',
+    'concerts',
+    'live',
+    'live shows',
+    'shows',
+    'gigs';
+
+spice proxy_cache_valid => "418 1d";
+spice is_cached => 0;
+
+spice to => 'http://api.seatgeek.com/2/events?taxonomies.name=concert&lat=$1&lon=$2&range=10mi&&callback={{callback}}';
```

Comment:
@MariagraziaAlastra Good catch, I didn't think of that - I guess I'm pretty lucky to live in a city with loads of venues :) Done.